### PR TITLE
Update: VoodooSoft.VoodooShield version 6.57

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20529)

### CODE REVIEW

File: manifests/v/VoodooSoft/VoodooShield/6.57/VoodooSoft.VoodooShield.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://voodooshield.com/Download/InstallVoodooShield.exe
```
Comment (ItzLevvie) [inline manifests/v/VoodooSoft/VoodooShield/6.57/VoodooSoft.VoodooShield.installer.yaml:16]:
```suggestion
  InstallerUrl: https://voodooshield.com/Download/InstallVoodooShield657.exe
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by quhxl** [comment]
#19757

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Looks like a PR was already open for it: https://github.com/microsoft/winget-pkgs/pull/20529#issuecomment-877874741.

**Comment by vedantmgoyal2009** [comment]
My PR passed but #19757 didn't. 🙂

**Review by OfficialEsco — APPROVED** [review]
Approving this one since it got more Metadata, direct download URL and passed validation multiple times for some reason

**Pull request merged**